nautobot/core/choices.py

```diff
@@ -43,8 +43,8 @@ class ChoiceSet(metaclass=ChoiceSetMeta):
     Example:
         ```python
         class GreekCapitalLetterChoices(ChoiceSet):
-            ALPHA = "Α"
-            BETA = "Β"
+            ALPHA = "A"
+            BETA = "B"
```

Comment:
Replacing sneaky Unicode Greek letters with their ASCII equivalents; could also have done a `# noqa: RUF01` if we wanted to keep them as-is.